.docker/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.22.1-alpine AS builder
+FROM golang:1.23.0-alpine AS builder
```

let's use the latest 1.23 image
```suggestion
FROM golang:1.23.9-alpine AS builder
```